lightstep/sdk/metric/exporters/otlp/internal/retry/retry_test.go

```diff
@@ -133,7 +131,9 @@ func TestBackoffRetry(t *testing.T) {
 	origWait := waitFunc
 	var done bool
 	waitFunc = func(_ context.Context, d time.Duration) error {
-		delta := math.Ceil(float64(delay)*backoff.DefaultRandomizationFactor) - float64(delay)
+		// N.B. the old formula was flaky, this code is EOL
```

Comment:
Here, EOL refers to the exporter code we've bulk copied from OTel-Go's SDK. I have pending work in this branch to decommission this exporter in favor of OTel Collector's OTLP exporter (and/or the OTel-Arrow exporter).